libr/core/cmd_print.c

```diff
@@ -686,9 +686,14 @@ static void cmd_prc(RCore *core, const ut8* block, int len) {
 			if (square) {
 				if (show_flags) {
 					RFlagItem *fi = r_flag_get_i (core->flags, core->offset + j);
-					if (fi) {
-						ch = fi->name[0];
-						ch2 = fi->name[1];
+					if (fi && fi->name && fi->name[0]) {
```

Comment:
use `!R_STR_ISEMPTY (fi->name)`

Reply:
btw, when can this happen? I think in theory this should never happen, as `name` should never be NULL nor empty.  Indeed function `r_flag_set` checks `r_return_val_if_fail (f && name && *name, NULL); `. I think that if you found some cases where this is not respected, there is an error somewhere else and we should fix that, not adding `if`s around.

Reply:
yes in theory it won't happen, both in practical use and with regard to the code, so 4d6bf841f6ed0a71bc09667aa252e6157d01ba29 should resolve this. Even if `name` was NULL, the worse that could happen is a segfault (loss of work) and that appears better than silent failure and possible output corruption.